### PR TITLE
[FAN-222] Add top creators component

### PR DESCRIPTION
**Pull request by Antroll**
https://fangible.atlassian.net/browse/FAN-222

### CODE REVIEW

**Review by sidorovkirill — APPROVED** [review]

File: src/modules/stories/components/StoriesOwner/StoriesOwnerSkeleton.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import { Typography } from '@material-ui/core';
+import { Skeleton } from '@material-ui/lab';
+import classNames from 'classnames';
+import React from 'react';
+import { useStoriesOwnerStyles } from './useStoriesOwnerStyles';
+
+interface IStoriesOwnerSkeletonProps {
+  className?: string;
```
Comment (BigBadAlien) [inline src/modules/stories/components/StoriesOwner/StoriesOwnerSkeleton.tsx]:
```suggestion

```

File: src/modules/stories/components/StoriesOwner/useStoriesOwnerStyles.ts [hunk 2]
```diff
@@ -39,4 +39,6 @@ export const useStoriesOwnerStyles = makeStyles<Theme>(theme => ({
       fontSize: 18,
     },
   },
+
+  titleThumb: {},
```
Comment (BigBadAlien) [inline src/modules/stories/components/StoriesOwner/useStoriesOwnerStyles.ts]:
```suggestion

```

File: src/modules/stories/screens/Stories/components/StoriesSlider/StoriesSliderComponent.tsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+        spaceBetween: theme.spacing(3.5),
+      },
+      [theme.breakpoints.values.xl]: {
+        slidesPerView: isFewSlides ? 2 : 1,
+        spaceBetween: theme.spacing(3.5),
+      },
+    },
+  };
+
+  const modifyChildren = (child: any) => (
```
Comment (BigBadAlien) [inline src/modules/stories/screens/Stories/components/StoriesSlider/StoriesSliderComponent.tsx]:
```suggestion
  const modifyChildren = (child: ReactNode) => (
```

File: src/modules/stories/screens/Stories/components/TopCreators/TopCreators.tsx [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+      setLoading(false);
+    }, 3000);
+  }, []);
+
+  const renderedSkeletons = new Array(7)
+    .fill(0)
+    .map((_, i) => <TopCreatorsItemSkeleton key={i} />);
+
+  const renderedItems = demoItems.map(({ img, title }, i) => (
+    <TopCreatorsItem href="#" key={uid(title, i)} img={img} title={title} />
```
Comment (BigBadAlien) [inline src/modules/stories/screens/Stories/components/TopCreators/TopCreators.tsx]:
```suggestion
    <TopCreatorsItem to="#" key={uid(title, i)} img={img} title={title} />
```

Comment (BigBadAlien) [reply src/modules/stories/screens/Stories/components/TopCreators/TopCreators.tsx]:
```
<Link component={TopCreatorsItem}  key={uid(title, i)} img={img} title={title} />
```

**Review by BigBadAlien — APPROVED** [review]

File: src/modules/stories/screens/Stories/components/TopCreators/TopCreators.tsx (hunk 4 above)
Comment (Antroll) [reply src/modules/stories/screens/Stories/components/TopCreators/TopCreators.tsx]: > ```jsx
> <Link component={TopCreatorsItem}  key={uid(title, i)} img={img} title={title} />
> ```

The `Link` component is not typed for this case.

**Pull request merged**